engine/table/src/main/java/io/deephaven/engine/table/impl/by/HashedRunFinder.java

```diff
@@ -72,7 +72,8 @@ public static boolean findRunsHashed(
 
         for (int chunkPosition = 0; chunkPosition < size; ++chunkPosition) {
             final int outputPosition = outputPositions.get(chunkPosition);
-            int hashSlot = outputPosition & context.tableMask;
+            // Increase the chance of hash collision, but reduce the linear probing needed in pathological cases.
```

Comment:
Increase relative to what? This comment is more of a commit message than an educational comment in the code. Might want to include the original code in the comment, and provide more context.